website/src/panel-layout.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { ReactNode } from "react"
+import { AiFillSound } from "react-icons/ai"
+import { GrDown, GrUp } from "react-icons/gr"
+import { IoEllipsisHorizontalCircle } from "react-icons/io5"
+import { MdClose, MdNotes, MdRecordVoiceOver } from "react-icons/md"
+import {
+  Disclosure,
+  DisclosureContent,
+  useDisclosureState,
+} from "reakit/Disclosure"
+import { unstable_Form as Form } from "reakit/Form"
+import { useCredentials } from "./auth"
+import { AudioPlayer, IconButton } from "./components"
+import EditWordPanel, { EditButton } from "./edit-word-panel"
+import { useForm } from "./form-context"
+import { FormFieldsFragment } from "./graphql/dailp"
+import * as css from "./panel-layout.css"
+import { WordPanel } from "./word-panel"
+
+export interface PanelDetails {
+  currContents: FormFieldsFragment | null
+  setCurrContents: (currContents: FormFieldsFragment | null) => void
+}
+
+/** Displays the right-side panel information of the currently selected word. */
+export const PanelLayout = (props: {
+  segment: FormFieldsFragment | null
+  setContent: (content: FormFieldsFragment | null) => void
+}) => {
+  if (!props.segment) {
+    return null
+  }
+
+  const { form, isEditing } = useForm()
+  const token = useCredentials()
+
+  return (
+    <div className={css.wordPanelContent}>
+      <header className={css.wordPanelHeader}>
+        <div className={css.headerButtons}>
+          <IconButton
+            onClick={() => props.setContent(null)}
+            aria-label="Dismiss selected word information"
+          >
+            <MdClose size={32} />
+          </IconButton>
+
+          {token && <EditButton />}
+        </div>
+
+        <h2 className={css.cherHeader}>{props.segment.source}</h2>
```

Comment:
The styling for this element is off, it shouldn't have the margin on the sides and it's got a big margin above it.

Reply:
I reverted the styling for this element, but kept this styling for when the panel contains an edit button if that's okay.